packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md

```diff
@@ -4,139 +4,105 @@ excerpt: >-
   Understand how to use the rate limit dashboard, how to access it, how to use the multiple views using the graphs, and how to investigate a rate limit violation
 ---
 
-# Client-based rate limits
+# Rate limit dashboard
 
-Client-based rate limiting for the `/authorize` endpoint uses a combination of the Client ID, user's IP address, and Okta device identifier to provide granular isolation for requests made to the `/authorize` endpoint. This framework isolates rogue OAuth clients and bad actors, thereby ensuring valid users and applications don't run into rate limit violations.
+The rate limit dashboard is a tool to help you to understand and investigate the rate limit and the current utilization of each endpoint. The dashboard provides:
```

Comment:
Utilization isn't conversational and is considered jargon. Can we switch that out, please?

utilization -> use